packages/amplication-server/src/core/auth/auth.controller.ts

```diff
@@ -182,6 +183,13 @@ export class AuthController {
       isNew = false;
     }
 
+    this.authService.trackCompleteEmailSignup(
+      user.id,
```

Comment:
You need to pass the Account.id rather than the User.id (or pass the entire user object and let the `trackCompleteEmailSignup` extract the account) since the identifier for the Analytics is always the Account.id and not the User.id

Reply:
done! Nice catch!